docs/lexical-search.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>You should mention here that if you do this reranking, the responses might not be ordered on on their lexical relevance along anymore. I.E, it's possible to have an ordering of lexical top to bottom, and that get's a bit muddled after reranking semantically, but relevance is maximized nonetheless.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>